src/crimson/os/seastore/seastore_types.h

```diff
@@ -1156,16 +1159,21 @@ struct extent_info_t {
   extent_types_t type = extent_types_t::NONE;
   laddr_t addr = L_ADDR_NULL;
   extent_len_t len = 0;
+  uint64_t last_modified = 0;
```

Comment:
Introduce a type in seastore_types.h for modification_time and document it.

Reply:
yes

Reply:
Include that in the comment for the time type.